indico/modules/events/persons/forms.py

```diff
@@ -38,3 +38,9 @@ def __init__(self, *args, **kwargs):
 
     def is_submitted(self):
         return super().is_submitted() and 'submitted' in request.form
+
+
+class ManagePersonListsForm(IndicoForm):
+    disallow_custom_persons = BooleanField(_('Disallow manually entering persons'), widget=SwitchWidget(),
+                                           description=_('Only allow submitters to list existing Indico users as '
```

Comment:
What about "Prohibit submitters from new speakers/authors manually and only allow searching for existing users."? That way it removes the focus from "indico users" ("users with no indico account" is a thing when searching e.g. the sso directory)